src/perftest_parameters.c

```diff
@@ -432,9 +432,9 @@ static void usage(const char *argv0, VerbType verb, TestType tst, int connection
 		printf(" Report RX & TX results separately on Bidirectinal BW tests\n");
 
 		printf("      --report_gbits ");
-		printf(" Report Max/Average BW of test in Gbit/sec (instead of MB/sec)\n");
-		printf("        Note: MB=2^20 byte, while Gb=10^9 bits. Use these formulas for conversion:\n");
-		printf("        Factor=10^9/(20^2*8)=119.2; MB=Gb_result * factor; Gb=MB_result / factor\n");
+		printf(" Report Max/Average BW of test in Gbit/sec (instead of MiB/sec)\n");
```

Comment:
I agree that MiB the correct term to use here, but it is not widely used and I am afraid it will lead to misunderstanding here. Lets see what other people think.

Reply:
I agree

Reply:
I agree that MiB is less known, however MB is well known and people are wondering why they are getting very different BW numbers in MB units while using different tools, including perftest. The motivation behind this change is to make it less confusing. Unless your users run only benchmarks which generate results in MiB while reporting them in MB, your users are also going to be confused by this.

Reply:
I prefer to leave it as it was.